app/Console/Command/ReplayCommand.php

```diff
@@ -240,14 +240,28 @@ private function askConfirmation(InputInterface $input, OutputInterface $output)
         $cdbids = $input->getOption(self::OPTION_CDBID);
 
         $message = 'Are you sure you want to replay all events? [y/N] ';
-        if ($aggregateType) {
-            $message = 'Are you sure you want to replay all events for aggregate type ' . $aggregateType->toString() . '? [y/N] ';
-        }
-        if ($startId) {
-            $message = 'Are you sure you want to replay all events starting from id ' . $startId . '? [y/N] ';
-        }
-        if ($cdbids) {
-            $message = 'Are you sure you want to replay all events with given cdbid? [y/N] ';
+
+        if ($aggregateType || $startId || $cdbids) {
+            $options = '';
+            if ($aggregateType) {
+                $options .= 'aggregate type: ' . $aggregateType->toString();
+            }
+
+            if ($startId) {
+                if (!empty($options)) {
+                    $options .=', ';
+                }
+                $options .= 'start id: ' . $startId;
+            }
+
+            if ($cdbids) {
+                if (!empty($options)) {
+                    $options .=', ';
+                }
+                $options .= 'given cdbids';
+            }
+
+            $message = 'Are you sure you want to replay events ( ' . $options . ' )? [y/N]';
```

Comment:
Nit: You can make `$options` an array of strings and do `implode(', ' $options)` here to concatenate them without the repeated `if (!empty($options)) { $options .=', '; }` checks.
(If you do an implode on an array with no values it will return an empty string, with 1 value it will return that value, and with multiple values it will concatenate them with the given "glue" in between every value)